src/pineko/theory.py

```diff
@@ -224,7 +224,7 @@ def opcard(self, name, grid, xif, tcard_path):
                 return
         _x_grid, q2_grid = evolve.write_operator_card_from_file(
             grid,
-            self.operator_cards_path / configs.configs["paths"]["operator_card_template_name"],
+            configs.configs["paths"]["operator_card_template"],
```

Comment:
huh? why did you revert to the old style?

Reply:
Because scaffolding is not yet merged so I would say let's keep the old style and then let's change once scaffolding is merged. Of course I can even do it now